packages/jsActions/native-mobile-resources/package.json

```diff
@@ -18,11 +18,12 @@
     "@react-native-community/cameraroll": "1.4.1",
     "react-native-device-info": "5.5.3",
     "react-native-firebase": "5.6.0",
-    "react-native-image-picker": "2.3.1",
+    "react-native-image-picker": "4.0.3",
```

Comment:
Needs an update in the main package-lock

Reply:
Same as https://github.com/mendix/widgets-resources/pull/668/files#r663687936

Reply:
Done